content/en/docs/getting-started/setup/create-cluster/google.md

```diff
@@ -7,8 +7,261 @@ publishdate: 2017-02-01
 lastmod: 2017-02-01
 weight: 50
 ---
-# Creating a Google cluster
 
+Google Cloud Platform (GCP) offers a powerful service for creating
+robust, highly-available clusters using the Kubernetes automated
+deployment platform. Google Kubernetes Engine (GKE) is a
+highly-available and secure kubernetes implementation that leverages the
+scalability of Google Cloud to provide a comprehensive solution when
+combined with the cloud-native development power and stability of
+Jenkins X.
+
+Cluster installation and configuration in Jenkins X is possible through
+*Jenkins X Boot*, a command that allows users to create Kubernetes
+clusters without running through various commands for configuring
+storage, pipeline, GitOps, and other resources.
+
+# Cluster prerequisites
+
+In order to create a cluster in Jenkins X, you must have the following
+configured:
+
+-   A Google Cloud Platform (GCP) account for creating kubernetes
+    clusters
+
+-   The Kubernetes command-line tool, which can be installed to your
+    local installation using the `jx install` command:
+
+        jx install dependencies -d kubectl
```

Comment:
What is the benefit of installing `kubectl` with that command as opposed to whatever is described in https://kubernetes.io/docs/tasks/tools/install-kubectl/?